swift/extractor/visitors/DeclVisitor.cpp

```diff
@@ -305,6 +305,8 @@ void DeclVisitor::fillAbstractFunctionDecl(const swift::AbstractFunctionDecl& de
   entry.name = !decl.hasName() ? "(unnamed function decl)" : constructName(decl.getName());
   entry.body = dispatcher_.fetchOptionalLabel(decl.getBody());
   entry.params = dispatcher_.fetchRepeatedLabels(*decl.getParameters());
+  auto self = const_cast<swift::ParamDecl* const>(decl.getImplicitSelfDecl());
```

Comment:
Can we have a TODO here? Something like "TODO: figure out const correctness" would work 😄

Reply:
Indeed. I'll add such a comment in another PR 👍.